pymc3/variational/advi.py

```diff
@@ -222,7 +216,7 @@ def run_adagrad(uw, grad, elbo, n, learning_rate=.001, epsilon=.1, verbose=1):
 
 def variational_gradient_estimate(
     vars, model, minibatch_RVs=[], minibatch_tensors=[], total_size=None, 
-    n_mcsamples=1):
+    n_mcsamples=1, seed=1):
```

Comment:
Interesting, you're correct. The default initialize is to `12345` (http://deeplearning.net/software/theano/library/sandbox/rng_mrg.html). In that case, we shouldn't randomize the seed but just use the same default.